src/main/java/com/microsoft/azure/functions/HttpResponseMessage.java

```diff
@@ -22,7 +22,7 @@
      * 
      * @return the status code set on the HttpResponseMessage instance.
      */
-    HttpStatus getStatus();
+    int getHttpStatus();
```

Comment:
Can we have both? Giving user a choice between using enums vs int

Reply:
We can't. HttpStatus enum is for standard http codes. If user sets a custom code, getStatus would have to return null, which is error prone in user code.

Reply:
Why not throw a runtime exception when the custom code is not in the standard HTTP codes?